src/args.rs

```diff
@@ -25,5 +25,7 @@ pub struct StatisticsSampleParameters {
     pub hands_number: usize,
 
     pub number_of_samples: u32,
+
+    pub number_of_threads: u32,
```

Comment:
Three positional integer arguments gets a bit hard to remember. Perhaps we should (as a follow-up PR) make each of these have an explicit name... could go either way. Fine to leave as-is.